revolv/templates/project/project.html

```diff
@@ -13,223 +14,344 @@
+                                                <div class="venue-map" id="map-canvas"></div>
+                                                <div class="venue-bottom">
+                                                    <h4 class="title-h4">{{ project.title }}</h4>
+                                                    <p>
+                                                        <span>{{ project.location_street }}</span>
+                                                        <span>{{ project.location_city_state_zip }}</span>
+                                                    </p>
+                                                </div>
+                                            </div>
+                                           
+                                        </div> -->
+                                        <!-- end .main-area -->
+                                        <!-- <div class="col-sm-4 pull-right">
+                                            <div class="right_asides">
+                                                <aside class="right-aside">
+                                                    <div class="module-box payment-module">
+                                                        <h1 class="basic">Gold</h1>
+                                                        <strong class="dollar">$ 80</strong>
+                                                        <span class="our_apps">Our Appritiation</span>
+                                                        <p class="dummy">Lorem ipsum Proim gravida module-box payment-module module-box payment-module module-box payment-module</p>
+                                                        <a href="/signin/?next=/project/1/&amp;reason=donate#login" id="donate-button" class="btn-blue btn-i-want-to-donate">
+                                                            <p>DONATE</p>
+                                                        </a>
+                                                    </div>
+                                                </aside>
+                                            </div>
+                                        </div> -->  
+                                        <!-- end .list-area -->
+                                        <div class="main-area1 col-sm-12">
+                                        <div class="container">
+                                            <!-- end .list-area -->
+                                            <div class="venue-area">
+                                                <h2 class="title-blue-border title-venue pull-left">DONORS</h2>
+                                        </div>
+                                        </div>
+                                    </div>
+                                    <div class="clearfix"></div>
 
-  <div class="donors-module">
+                                    <!-- tabs start here -->
```

Comment:
This is a great comment, however, please de-indent it to match the indentation level of the next line.